app/assets/stylesheets/custom.css.scss

```diff
@@ -80,3 +80,22 @@ li.active > a
 .btn-width{
     width: 150px;
 }
+
+li {
+  padding-top: 8px;
+  padding-bottom: 12px;
+}
+#file {
+  color: purple;
```

Comment:
Color `purple` should be written in hexadecimal form as `#800080`<br>Color literals like `purple` should only be used in variable declarations; they should be referred to via variable everywhere else.